src/objective-c/GRPCClient/GRPCCall.m

```diff
@@ -103,6 +103,11 @@ @implementation GRPCCall {
 
 @synthesize state = _state;
 
+// TODO(jcanizales): If grpc_init is idempotent, this should be changed from load to initialize.
++ (void)load {
```

Comment:
Then we need to change all the examples and tests to invoke load() before they can run properly.